src/clojisr/v1/r.clj

```diff
@@ -30,8 +30,10 @@
   (let [session (session/fetch-or-make session-args)]
     (prot/eval-r->java session r-code)))
 
-(defn r->java [r-object]
-  (using-sessions/r->java r-object))
+(defn r->java [r-object & options]
+  (using-sessions/r->java 
+   (assoc r-object :options options)
```

Comment:
RObject is defined by `defrecord`, since you can assoc to a record it's not a good way of doing it. I think the proper way would be extend record for options.
Anyway why assigned it here?

Reply:
not sure any more, what this does

Reply:
So from my perspective it's not necessary.

Reply:
yes, I will remove teh options for r->java.
And then I make a first trial of "convert" to tensor when a certain option is set.
Somehwhere here, I suppose:
https://github.com/scicloj/clojisr/blob/625822f388b0d18e015f2e9999e09f91c05cbe2b/src/clojisr/v1/impl/java_to_clj.clj#L164